nncf/experimental/common/quantization/algorithms/post_training/algorithm.py

```diff
@@ -0,0 +1,106 @@
+# Copyright (c) 2024 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import itertools
+from typing import Callable, List, Optional, TypeVar
+
+from nncf import Dataset
+from nncf.common.graph.graph import NNCFGraph
+from nncf.common.tensor_statistics.statistic_point import StatisticPointsContainer
+from nncf.common.utils.backend import BackendType
+from nncf.experimental.common.quantization.algorithms.post_training.pipeline import experimental_create_ptq_pipeline
+from nncf.experimental.common.quantization.algorithms.quantizer.base_quantizer import NNCFQuantizer
+from nncf.quantization.advanced_parameters import AdvancedBiasCorrectionParameters
+from nncf.quantization.advanced_parameters import AdvancedSmoothQuantParameters
+from nncf.quantization.advanced_parameters import RangeEstimatorParameters
+from nncf.quantization.algorithms.algorithm import Algorithm
+
+TModel = TypeVar("TModel")
+TPass = Callable[[TModel], TModel]
+
+
+class ExperimentalPostTrainingQuantization(Algorithm):
+    """
+    Implements Experimental Post-Training Quantization algorithm, which basically includes:
+    1) ChannelAlignment
+    2) MinMaxRangeInit
+    3) FastBiasCorrection or BiasCorrection
+    """
+
+    def __init__(
```

Comment:
Please, align `__init__` method with regarding my comments to the `quantize_pt2e` function.

Reply:
Please share the link with your comments, I could not find it there

Reply:
I decide the the value of `batchwise_statistics` during the `quantize_pt2e`, so future values are booleans